langlib/lang.test/src/main/ballerina/src/lang.test/testfunctions.bal

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
The convention for bal file names is snake case (with lower case). 

Shall we rename this as `test_functions.bal`?